src/js/game/LevelMVC/LevelView.js

```diff
@@ -2015,6 +2025,102 @@ module.exports = class LevelView {
     return sprite;
   }
 
+  determineRedstoneSprite(x, y, blockType, levelData) {
+        let foundAbove = false;
+        let foundBelow = false;
+        let foundRight = false;
+        let foundLeft = false;
+        let belowIndex = (this.yToIndex(y + 1)) + x;
+        let aboveIndex = (this.yToIndex(y - 1)) + x;
+        let leftIndex = (this.yToIndex(y)) + x - 1;
+        let rightIndex = (this.yToIndex(y)) + x + 1;
+
+        let borderCount = 0;
+
+        //need to ensure these are in bounds
+        if (y === levelData.planeHeight) {
```

Comment:
Please use `levelModel.inBounds`.  Rather than referencing it through the controller, I think it would make more sense for this function to live in `LevelModel.js`.